it/test/connectors/WireMockHelper.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.github.tomakehurst.wiremock.WireMockServer
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig
+import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach, Suite}
+
+trait WireMockHelper extends BeforeAndAfterAll with BeforeAndAfterEach {
+  this: Suite =>
+  protected val server: WireMockServer = new WireMockServer(wireMockConfig().dynamicPort())
```

Comment:
Does .dynamicPort sort all the config out automatically? If not you need to override your config here and set your outbound ports to the same one the WireMockServer is running on

Reply:
Ah just noticed you were doing this in the spec itself so ignore this

Reply:
@simondrugan16  - I based this on your returns FE. 

The port is overridden at the top of the connector spec. 

<img width="906" alt="Screenshot 2024-11-21 at 11 24 49" src="https://github.com/user-attachments/assets/ce550121-4c89-4d8e-a791-8f18feb5935e">

Reply:
@simondrugan16  -  I am keen to use the same or similar code / test style in the new services such that they are easy to follow for new dev's getting to grips with the entire suite.